tsconfig.json

```diff
@@ -0,0 +1,72 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "ES2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```

Comment:
IMO for a node SDK it's still a good idea to just target ES5

Reply:
We are only supporting node >=10 

We don't need to compile before ES2020 as far as I know.

Reply:
nope I have that wrong, we want ES2018

Reply:
yes 2020, will compile with `import/export` syntax which won't work in node 10

Reply:
`"module": "commonjs"` will make it compile to `module.exports` and `require()` but there are other syntax changes that will not be supported but node 10.

Changing now.